spec/components/auth/instagram_authenticator_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+describe Auth::InstagramAuthenticator do
+
+  it "takes over account if email is supplied" do
+    auth = Auth::InstagramAuthenticator.new
+
+    user = Fabricate(:user)
```

Comment:
Can we move this to before to a `let(:user)` above the `it ..`, I can see we are using another `fabricate(:user)` [below](https://github.com/discourse/discourse/pull/7081/files#diff-431c322d77be415c017de4b9f5567c9bR43).

Reply:
I'm kind of reluctant to do this, I followed the same patterns we have in related tests files for other authenticators